### DISCUSSION TITLE
[Bug] Unclear Documentation: scoop update before update * ?

### DISCUSSION THREAD

**Discussion by falco467** [ScoopInstaller/Scoop#5290] [Ideas]
#### Current Behavior
The documentation is not absolutely clear if `scoop update *` will be enough to keep all my apps (including scoop) up to date, or if I regularly have to run `scoop update` too.

#### Expected Behavior
I would like for `scoop update *` to be everything I need to run regularly. Or if this is not enough maybe an argument which will execute the additional steps of `scoop update` before `update *` if needed.

#### Additional context/output
I've read in multiple discussions people being unsure if `scoop update *` is enough. So the documentation should be absolutely clear if/when `scoop update` is additionally needed.

#### Possible Solution
Best solution `scoop update *` is enough and amend the documentation to clearly state this.
If there is a reason this should not be the default, maybe provide an argument like `scoop update * --update-scoop` so we have a single command.

If this is also not possible at least amend the documentation with a clear warning about `update *` not updating scoop and `scoop update` always being needed additionally.

**Comment by rashil2000** [comment]
Running `scoop update *` should be enough. 

Could you point out where the documentation is unclear?

**Reply by falco467** [reply]
https://scoop-docs.vercel.app/docs/getting-started/Quick-Start.html#updating-scoop

Here it says

To get the latest version of Scoop you have to run the command: `scoop update`
This will download the latest version of scoop and updates the local app manifests.
After you updated Scoop you can update individual apps: `scoop update curl`

If you want to update all your installed apps, you can run: `scoop update *`

---

This implies that I will need to run `scoop updated` to get the latest version of scoop, since `scoop update *` only updates "all your installed apps"

Maybe just add an addendum directly after `scoop update *`: "This will also download the latest version of scoop and app manifests before updating all your installed apps."

**Reply by niheaven** [reply]
"All your installed apps" includes scoop itself, in any means. `scoop update` will be run by `scoop update *` every 4 hrs now.

**Reply by falco467** [reply]
Thank you! I think this bit of information (scoop update * will regularly run scoop update) could be added to the documentation.

For example in this thread about scheduled updates multiple users believe they need to run update before update * https://github.com/ScoopInstaller/Scoop/issues/3633

**Reply by niheaven** [reply]
Okay, gotta, will changed soon.

**Reply by niheaven** [reply]
Changed now in wiki.
